cmd/admin-handlers.go

```diff
@@ -49,6 +52,8 @@ import (
 	trace "github.com/minio/minio/pkg/trace"
 )
 
+var errNotConnected = errors.New("not connected to target server/service")
```

Comment:
This error definition is ambiguous. We already have a definition in target package. And since etcd is not a target the error message can be `not connected to etcd endpoint`